src/tests/ftest/erasurecode/online_rebuild.yaml

```diff
@@ -65,4 +65,5 @@ ior:
       #- [EC_Object_Class, Minimum number of servers]
       - ["EC_2P2G1", 6]
       - ["EC_4P2G1", 8]
+      - ["EC_4P3G1", 9]
```

Comment:
shouldn't that be 10 instead of 9?

Reply:
Although the comment says "Minimum number of servers", the code implements "Exact number of servers" because it uses `==`
https://github.com/daos-stack/daos/blob/7d3241b4933e0696b8fc796909c26f07b0ca6618/src/tests/ftest/erasurecode/online_rebuild.py#L43
So we should actually set this to `12` so the test runs with 4P3 at all.
(because above in this file there is 6, 8, 12 server variants)

Reply:
And please update the comment above :)

Reply:
just note 4P3G1 with 7 shards per object. for placement consideration 7 engines is enough, may need to add a few more spare engines if kills some ranks in the test.

Reply:
this is supposed to survive 3 engine failures. so we need 3 spares.